### PR TITLE
fix new config map names

### PR DESCRIPTION
**Pull request by mjudeikis**
move osa jobs to new sharded config map names

/cc @jim-minter @Makdaam @petr-muller

### CODE REVIEW

**Comment by petr-muller** [comment]
/hold

mmnt too trigger-happy

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ periodics:
         valueFrom:
           configMapKeyRef:
             key: openshift-openshift-ansible-release-3.11.yaml
-            name: ci-operator-configs
+            name: ci-operator-master-configs
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: This looks for a 3.11 config so this should reference `ci-operator-3.x-configs` CM

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 2]
```diff
@@ -124,7 +124,7 @@ periodics:
...
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: Ditto

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml [hunk 3]
```diff
@@ -171,7 +171,7 @@ periodics:
...
```
Comment (petr-muller) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml]: Ditto

**Comment by mjudeikis** [comment]
@petr-muller fixed

**Pull request merged**